atd-etl/cris_import/cris_import.py

```diff
@@ -112,7 +103,8 @@ def main():
             align_records_token = align_records(typed_token)
             clean_up_import_schema(align_records_token)
         mark_extract_as_imported(archive_id)
-        move_extract_into_processed(filename_in_s3)
+        if not local_mode:
+            move_extract_into_processed(filename_in_s3)
```

Comment:
This function expects the file we're operating on to have come from S3 in production, which is not the case in dev mode.